### DISCUSSION TITLE
Current best practice for transfer of ERC721 ownership

### DISCUSSION THREAD

**Discussion by 20skcs** [smartcontractkit/full-blockchain-solidity-course-py#974] [General]
Interested in hearing thoughts on what is considered current best practice (in relation to security/reentrancy) for the transferring of ownership of an erc721 in a sale situation (you are the owner and you are interacting with a buyer)? There is conflicting information online and the most recent relevant tutorial example I could find on Ethereum was from early 2020.

**Comment by cromewar** [comment]
Hello @20skcs 

1. Fist of all, is important the contract to be verified on a Block explorer like Etherscan, there have been a lot of scams with fake ERC721 tokens which withdraw all the receiver balance, so if the contract is verified is even secure.
2. If you want to go deep into security, paying a smart contract audit company also is good.
3. Never try to trade any NFT on your wallet which you are not sure is yours (as I said there are a lot of scams out there).
4. Never buy NFTs from a website you don't trust, check if the project has a twitter community, discord server and etc.

Summarizing, programmatically the best way to ensure security on this field is always verify the contracts on a block explorer. And on the user related stuff, always be cautious.

I know is a shame, but there is not something like a worldwide defined standard on this, there is something people on the space are working on , or ignoring it completely.
